DYDX-Bots/Operator-monitor/src/agent.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { Interface } from "ethers/lib/utils";
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+} from "forta-agent";
+import {
+  createAddress,
+  TestTransactionEvent,
+} from "forta-agent-tools/lib/tests";
+import { provideHandleTransaction } from "./agent";
+import { MONITORED_EVENTS } from "./agent";
+
+// function to generate test findings
+const createFinding = (name: string, operator: string) => {
+  const description = name === "LogOperatorAdded" ? "added to" : "removed from";
+  const alertId = name === "LogFrozen" ? "DYDX-4-1" : "DYDX-4-2";
```

Comment:
```suggestion
  const alertId = name === "LogOperatorAdded ? "DYDX-4-1" : "DYDX-4-2";
```
Same suggestion as `agent.ts`